src/Symfony/Component/Translation/Bridge/PoEditor/PoEditorProvider.php

```diff
@@ -0,0 +1,204 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Translation\Bridge\PoEditor;
+
+use Psr\Log\LoggerInterface;
+use Symfony\Component\Translation\Exception\ProviderException;
+use Symfony\Component\Translation\Loader\LoaderInterface;
+use Symfony\Component\Translation\Provider\ProviderInterface;
+use Symfony\Component\Translation\TranslatorBag;
+use Symfony\Component\Translation\TranslatorBagInterface;
+use Symfony\Contracts\HttpClient\HttpClientInterface;
+
+/**
+ * @author Mathieu Santostefano <msantostefano@protonmail.com>
+ *
+ * @experimental in 5.3
+ *
+ * In PoEditor:
+ *  * Terms refer to Symfony's translation keys;
+ *  * Translations refer to Symfony's translated messages;
+ *  * Tags refer to Symfony's translation domains
+ *
+ * PoEditor's API always returns 200 status code, even in case of failure.
```

Comment:
🎉 😄

Reply:
Fortunately, in all responses body, there is something like 
```
[
    'response' => [
        'status' => 'success',
        'code' => '200',
        'message' => 'OK',
    ],
]
```
:sweat_smile: